HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -1680,8 +1682,9 @@ def self.adjust_sizing_by_installation_quality(hvac_final_values, weather, hvac,
         heat_cap_fff = (p_CH_Q_h * p_AF_Q_h)
         heat_cap_fff_pre = a1_AF_Qgr_h + a2_AF_Qgr_h * ff_AF_h_pre + a3_AF_Qgr_h * ff_AF_h_pre * ff_AF_h_pre
         cap_htg_ratio = 1 / (heat_cap_fff / heat_cap_fff_pre)
-        hvac.CapacityRatioHeating[speed] *= cap_htg_ratio
```

Comment:
I was trying to increase the capacity for each speed by using capacity ratio, but it doesn't seem as good as directly increasing capacity, considering the logic in process_fixed_equipment.